conftest.py

```diff
@@ -0,0 +1,26 @@
+""" This file configures python logging for the pytest framework
+integration tests
+
+Note: pytest must be invoked with this file in the working directory
+E.G. py.test frameworks/<your-frameworks>/tests
+"""
+import logging
+import os
+
+
+log_level = os.getenv('TEST_LOG_LEVEL', 'INFO').upper()
+
+log_levels = (
+    'DEBUG',
+    'INFO',
+    'WARNING',
+    'ERROR',
+    'CRITICAL',
+    'EXCEPTION')
+
+assert log_level in log_levels, '{} is not a valid log level. ' \
+    'Use one of: {}'.format(log_level, ', '.join(log_levels))
+
+logging.basicConfig(
+    format='[%(asctime)s|%(name)s|%(levelname)s]: %(message)s',
```

Comment:
This already has `name` which is the module name (i.e. the filename). I think line number is a bit overkill as long as you add any kind of context to your log statements and can therefore search for them. This is a bit crowded with the full acstime, so I think I want to leave this as is